pom.xml

```diff
@@ -83,4 +83,32 @@
             </modules>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M3</version>
+                <executions>
+                    <execution>
+                        <!--
+                            Enforce minimum of maven 3.5.4
+                        -->
+                        <id>enforce-maven</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireMavenVersion>
+                                    <version>3.5.4</version>
+                                </requireMavenVersion>
```

Comment:
3.5.4 is OLD (from 2018) but I think that's the version that's on the build nodes.

Reply:
When we're feeling brave we can try to jump to 3.8.x

Reply:
At least this provides a decent baseline to start with